src/App.js

```diff
@@ -33,6 +33,8 @@ const useStyles = makeStyles((theme) =>
     faceCaptureWrapper: {
       borderRadius: 20,
       overflow: "hidden",
+      paddingTop: "3vh",
+      paddingBottom: "3vh",
```

Comment:
Does it break the landscape view?

Reply:
I've rechecked and I've seen an issue with the borders that I didn't see before 😅, so I am going to set the PR as a draft until I fix it.

Reply:
All fixed and explained [here](https://github.com/getyoti/web-fcm-demo/pull/36#issuecomment-2163114115).